modules/ui_patterns_field_group/src/Plugin/field_group/FieldGroupFormatter/PatternFormatter.php

```diff
@@ -99,6 +99,9 @@ public function preRender(&$element, $rendering_object) {
     // Pass current entity to pattern context, if any.
     if (!empty($element['#fields'])) {
       $field = reset($element['#fields']);
+      if (!empty($field)) {
+        $field = reset($field);
```

Comment:
Why exactly is this needed? I can't get my head around this, could you post an example?